jetstream/stream_config.go

```diff
@@ -26,154 +26,356 @@ import (
 type (
 	// StreamInfo shows config and current state for this stream.
 	StreamInfo struct {
-		Config  StreamConfig        `json:"config"`
-		Created time.Time           `json:"created"`
-		State   StreamState         `json:"state"`
-		Cluster *ClusterInfo        `json:"cluster,omitempty"`
-		Mirror  *StreamSourceInfo   `json:"mirror,omitempty"`
+		// Config contains the configuration settings of the stream, set when
+		// creating or updating the stream.
+		Config StreamConfig `json:"config"`
+
+		// Created is the timestamp when the stream was created.
+		Created time.Time `json:"created"`
```

Comment:
there is also a `ts` in the stream info that shows the time the server created this info and you can use that to deal with clock skews between client and server, not sure if its worth surfacing but I use it all the time in cli

Reply:
I guess if you're using it in CI all the time then there are valid use cases for it. I will add it in a separate PR for clarity.

Reply:
I use it to calculate correct deltas like "Created 1m ago" or similar, but if my local clock skews I cant do time.Since(Created) so i calculate the age relative to ts+created, essentially using the servers clock not the clients clock.